### PR TITLE
Support partial style type

### PR DESCRIPTION
**Pull request by IIsaku**

### CODE REVIEW

File: generators/app/templates/pluggable/native/emptyTemplateTs/src/utils/common.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export interface Style {
     [key: string]: CustomStyle | ViewStyle | TextStyle | ImageStyle;
 }
 
-export function flattenStyles<T extends Style>(defaultStyle: T, overrideStyles: Array<(T | undefined)>): T {
+export function flattenStyles<T extends Style>(defaultStyle: T, overrideStyles: Array<(Partial<T> | undefined)>): T {
```
Comment (diego-antonelli) [inline generators/app/templates/pluggable/native/emptyTemplateTs/src/utils/common.ts:11]: Please add the same method to fullTemplateTS

**Review by diego-antonelli — CHANGES_REQUESTED** [review]

File: generators/app/templates/pluggable/native/emptyTemplateTs/src/utils/common.ts (hunk 1 above)
Comment (IIsaku) [reply generators/app/templates/pluggable/native/emptyTemplateTs/src/utils/common.ts:11]: Done

**Comment by IIsaku** [comment]
@diego-antonelli The tests are passing when using the Github dependencies.

**Review by diego-antonelli — APPROVED** [review]
Lgtm

**Pull request merged**